### ISSUE TITLE
Ignore trailing whitespace in a heredoc comment

### ISSUE THREAD

**Issue by jclem** [rrrene/credo#315]
Two trailing spaces are used in Markdown to indicate a line break.

The following heredoc:

```elixir
@doc """
Foo{space}{space}
Bar
"""
```

Would be rendered as:

```
Foo
Bar
```

Without the spaces, it would be rendered as:

```
Foo Bar
```

Currently, Credo warns "There should be no trailing white-space at the end of a line." when this valid Markdown is present in a heredoc comment.

**Comment by NobbZ** [comment]
Totally independant from the markdown semantics. Inside of quotes every
single char is meant as is and there should be no linting in general I
think.

Does `\s` work as a workaround until this gets fixed? And if so, I think
this might be even the better option. By writing `\s` explicitely, you make
sure that this is really what you wanted and not some typo.

Jonathan Clem <notifications@github.com> schrieb am Fr., 3. Feb. 2017 um
22:09 Uhr:
